Example/App/Sources/Networking/Interceptors/StatusCodeLoggingInterceptor.swift

```diff
@@ -0,0 +1,25 @@
+// Copyright © 2022 Jamit Labs GmbH. All rights reserved.
+
+import Foundation
+import Jetworking
+
+/// This is an example of an interceptor that specifically logs when one specific status code occurs.
+public final class StatusCodeLoggingInterceptor: Interceptor {
+    // MARK: - Properties
+    private var logger: Logger
```

Comment:
Can be defined as a constant?

```suggestion
    private let logger: Logger
```